plugins/modules/content_import_info.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2024, Partha Aji
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: content_import_info
+version_added: 4.0.0
+short_description: List content imports
+description:
+    - List information about content imports.
+author:
+    - "Partha Aji (@parthaa)"
+options:
+  id:
+    description:
+      - Export history identifier.
```

Comment:
Is this an export history or an import history? Are they the same?

Reply:
It should be import